shared/kubernetes/watchEvent.go

```diff
@@ -16,3 +18,12 @@ func (we *WatchEvent) ToPod() (*corev1.Pod, error) {
 
 	return pod, nil
 }
+
+func (we *WatchEvent) ToEvent() (*eventsv1.Event, error) {
+	pod, ok := we.Object.(*eventsv1.Event)
+	if !ok {
+		return nil, fmt.Errorf("Invalid object type on pod event stream")
```

Comment:
maybe change to something that is not similar to the error we get on ToPod

Reply:
done